### PR TITLE
[CUR-185] Fix examboard slug

### PR DESCRIPTION
**Pull request by sonalisin**
## Description

- Added fixtures
- Updated data for each page / component
- Pulled in correct slug to curriculum header 
- Updated tests

## How to test

1. Go to https://deploy-preview-1797--oak-web-application.netlify.thenational.academy/beta/teachers/curriculum/
2. Click on Secondary, English and then Edexcel
3. You should see `/english-secondary-edexcel` in the browser as the slug

## Checklist

- [ ] Added or updated tests where appropriate
- [ ] Manually tested across browsers / devices
- [ ] Considered impact on accessibility
- [ ] Design sign-off
- [ ] Approved by product owner

### CODE REVIEW

File: src/node-lib/curriculum-api-2023/queries/curriculumHeader/curriculumHeader.query.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    const subject = {
+      title: capitalise(subjectSlug),
+      slug: subjectSlug,
+    };
+    const phase = {
+      title: capitalise(phaseSlug),
+      slug: phaseSlug,
+    };
+    const examBoard = {
+      title: examBoardSlug ? capitalise(examBoardSlug) : "",
```
Comment (rightsaidjames) [inline src/node-lib/curriculum-api-2023/queries/curriculumHeader/curriculumHeader.query.ts:27]: @sonalisin This doesn't work for exam boards that are acronyms (e.g. AQA becomes `Aqa` in the breadcrumb):

**Review by rightsaidjames — CHANGES_REQUESTED** [review]
Original slug issue is fixed, but the capitalisation of certain exam boards is wrong.

File: src/node-lib/curriculum-api-2023/queries/curriculumHeader/curriculumHeader.query.ts (hunk 1 above)
Comment (sonalisin) [reply src/node-lib/curriculum-api-2023/queries/curriculumHeader/curriculumHeader.query.ts:27]: Yep - this was just placeholder - data and proper exam board info will be coming from the backend!

**Review by rightsaidjames — APPROVED** [review]

**Pull request merged**